TCIABrowser/TCIABrowser.py

```diff
@@ -358,10 +358,10 @@ def setup(self):
     seriesVBoxLayout2 = qt.QVBoxLayout(seriesExpdableArea)
     self.seriesTableWidget = qt.QTableWidget()
     # self.seriesModel = qt.QStandardItemModel()
-    self.seriesTableWidget.setColumnCount(12)
+    self.seriesTableWidget.setColumnCount(13)
     self.seriesTableWidget.sortingEnabled = True
     self.seriesTableWidget.hideColumn(0)
-    self.seriesTableHeaderLabels = ['Series Instance UID', 'Status', 'Series Description', 'Series Number', 
+    self.seriesTableHeaderLabels = ['Series Instance UID', 'Status', 'Patient ID', 'Series Description', 'Series Number', 
```

Comment:
In the series table I think we also need to add the Study Date now so that if multiple studies were selected you can tell which series came from which study in the series table.

Reply:
I wanted to add the Study Date as well in the Series table, but unfortunately it is not one of the metadata fields readily available from the geSeries() function. I'll check if there are other workaround but just a heads up, the implementation could be potentially complex.

Reply:
The output of getSeries includes a column called SeriesDate.  Let's just use that.  It *should* be the same as Study Date.

Reply:
Not all series have a StudyDate column, for example, it is "Time Stamp" 4D-Lung. I thought maybe there is a way to pass the study date from the Study table to the series table.

Reply:
Scratch that off, I believe most series, if not all, have the TimeStamp column, which actually is a different thing than the StudyDate. I will keep looking for a way to keep the StudyDate column from the Study table.

Reply:
Ah, ok.  Glad you checked.  I didn't realize that sometimes SeriesDate wasn't populated.  The Time Stamp column reflects the date that the data were "released" on TCIA, which is totally different than Study/Series Date (the anonymized date that the patient was scanned).

Reply:
I guess the other way to deal with this would be to display the Study Instance UID in both the Study table and the Series table.  That would allow users to see which series are part of which studies when viewing the aggregated list in the Series table.